test.js

```diff
@@ -88,7 +88,7 @@ test('children trigger notification', function(t) {
 
 test('autoloadJs hostname defaults to localhost', function(t) {
```

Comment:
This test description needs to be updated to match the logic of the test

Reply:
Oh, that's right.